src/modules/Builder/components/ItemFlowSelectController/ItemFlowSelectController.styles.ts

```diff
@@ -21,6 +21,7 @@ export const StyledSelectController = styled(SelectController)({
 
   '&& .MuiSelect-select': {
     padding: theme.spacing(0.65, 1.2),
+    minWidth: '14rem',
```

Comment:
Now that you added `width: 100%;` to `StyledTextField in `SelectController` I believe this is no longer necessary and will address @LashaunnaS' comment about smaller screens

| 14rem | 100% |
|-|-|
|![CleanShot 2024-10-30 at 16 07 45](https://github.com/user-attachments/assets/49dfdb2f-71eb-4667-81ef-5aaefde63f38)|![CleanShot 2024-10-30 at 16 07 32](https://github.com/user-attachments/assets/d544bf99-3b6e-4c0c-9a29-c460b100ebcc)|

Reply:
Nice spotting